### ISSUE TITLE
[Feature Request] Add convenience function to access the text value of a CompositeCell

### ISSUE THREAD

**Issue by kMutagene** [nfdi4plants/ARCtrl#190]
**Is your feature request related to a problem? Please describe.**
See https://github.com/nfdi4plants/ARCtrl/pull/189/files#r1323416055 , just tracking here so it is not forgotten.

**Comment by Freymaurer** [comment]
Is this what you want?

```fsharp
override this.ToString() = 
        match this with
        | Term oa -> $"{oa.NameText}"
        | FreeText s -> s
        | Unitized (v,oa) -> $"{v} {oa.NameText}"
```

**Comment by Freymaurer** [comment]
> just the value in the cell

What do you mean by this. Can you give some examples?

**Comment by kMutagene** [comment]
What is linked in above comment thread, the result of

```fsharp
cell.ToFreeTextCell().AsFreeText
```

**Comment by kMutagene** [comment]
> The only difference between both approaches is for the unitized case where you will also get the actual value.

yes, and this string `$"{v} {oa.NameText}"` cannot be used in my function.

I think it is even worse though, because the current code does return a wrong result for units, where i actually would need the value instead of NameText(although not really a problem as i do nit think paths usually have a unit).

A bit more detail:
- FreeText is easy as it models a single cell right?  so there it is just  `| FreeText s -> s`
- Term is the triple of this right?
   | cellHeader | TSR | TAN |
   |---|---|---|
   | **then gimme this** | not this | not this |
- Unitized is this?
   | cellHeader | unit header| unit TSR| unit TAN |
   |---|---|---|---|
   | **then gimme this** | not this | not this | not this |

**Comment by Freymaurer** [comment]
So you are interested and what you would see in the old Excel version in the main column. Then i wold recommend the ToString() function as this replicates this behaviour. If you need just the main column we can add a function. 👍

**Comment by kMutagene** [comment]
> So you are interested and what you would see in the old Excel version in the main column. Then i wold recommend the ToString() function as this replicates this behaviour. If you need just the main column we can add a function. 👍

I cannot use ToString as i pointedf out this combines value and unit, i just need value.

**Comment by kMutagene** [comment]
```fsharp
        match this with
        | Term oa -> $"{oa.NameText}"
        | FreeText s -> s
        | Unitized (v,oa) -> $"{v}"
```

i think this would be it, but would need to test

**Comment by Freymaurer** [comment]
Ah okay I have probably weighted that too much: "`i think (although not really a problem as i do not think paths usually have a unit).`"

**Comment by Freymaurer** [comment]
As this was not perfect to design, i chose a generally useful function:

```fsharp
member this.GetContent() = 
        match this with
        | FreeText s -> [|s|]
        | Term oa -> [| oa.NameText; defaultArg oa.TermSourceREF ""; defaultArg oa.TermAccessionNumber ""|]
        | Unitized (v,oa) -> [| v; oa.NameText; defaultArg oa.TermSourceREF ""; defaultArg oa.TermAccessionNumber ""|]
```

In you case you can run `cell.GetContent().[0]` and it is possible to match on arrays for cell specific workflows. In addition the return values are exactly what you would need to create the cells from strings.

Closed with #200
